src/spikeinterface/core/sorting_tools.py

```diff
@@ -197,17 +197,23 @@ def random_spikes_selection(
         cum_sizes = np.cumsum([0] + [s.size for s in spikes])
```

Comment:
Should we think about the possibility of overflow using the `np.cumsum` here? Max number of spikes for int64 is
9,223,372,036,854,775,807, which should give us a lot of leeway. Even just doing a back of the envelope isn't does seem like we would hit the limit yet, but it is worth keeping in mind that if we have  enough spikes this could overflow.

Reply:
Here it is the spikes indices not the traces shape.
It would be a good thing but I not sure we will handle 10e19 spikes very soon.

Reply:
Heberto and I will demand one saucisson/per overflow :P